### ISSUE TITLE
Blurry notification/Hi-DPI

### ISSUE THREAD

**Issue by ewwhash** [emersion/mako#387]
Using mako i got very strange behaviour: all the time i receive notifications, first notification gets blurry, but reloading sway/using slurp/getting another notification and all the notifications are no longer blurry.
![20211006_04h18m49s_grim](https://user-images.githubusercontent.com/28766913/136125715-3e637b6d-920b-4d9d-947c-d7040e2fbca5.png)
![20211006_04h19m00s_grim](https://user-images.githubusercontent.com/28766913/136125723-192ab8bd-13b1-4d33-81d3-72a4cadcdd58.png)

I'm using a Hi-DPI laptop, and of course with `output eDP-1 scale 1` everything works fine.

**Comment by BrightYC** [comment]
I don't know how can it help but i replaced `scale` to constant (2) in wayland.c and everything renders perfectly.

**Comment by trobjo** [comment]
Yes, that is true, even with your "fix" the cursor is still blurry when hovering over the notification.

**Comment by lilydjwg** [comment]
Today I've got really weird behavior:

1. restart mako
2. `notify-send` a message. it works as expected (shown, hidpi)
3. click to dismiss
4. `notify-send` another message. nothing appears
5. `makoctl reload`. the notification appears, but blurry.
6. issue some more reloads. it *probably* will become hidpi.

The two issues just seem to happen to me randomly. No clue what to do next.

**Comment by lilydjwg** [comment]
I've sent a pull request to address both blurry text and cursor. Could you give it a try?

**Comment by trobjo** [comment]
I have tested it now. You pull request fixes the issue for me!

**Comment by OctopusET** [comment]
I have tested it too, your PR fixes this issue!

**Comment by okhsunrog** [comment]
Created an AUR package for Arch Linux users
https://aur.archlinux.org/packages/mako-no-blur-git

**Comment by raiguard** [comment]
Thanks for the package @okhsunrog! Works like a charm.

**Comment by yavko** [comment]
Will is be merged?

**Comment by emersion** [comment]
Latest master should have this issue fixed.

**Comment by yavko** [comment]
> Latest master should have this issue fixed.

they are blurry again for some reason, and not just like sometimes but like every time now lol
